sig-ops/README.md

```diff
@@ -0,0 +1,27 @@
+# SIG: Ops
+
+This Special Interest Group (SIG) main goal is to coordinate Operations (Ops) activities and best practices across Spinnaker Operators.
+* Production Readiness
+* Security Requirements
+* Reference Architecture(s)  
+
+## Meetings
+
+* Regular SIG Meeting
+  * Every other TBD
+  * [Agenda]
+
+## Leadership
+
+* Edgar Magana [emagana](https://github.com/emagana))
```

Comment:
We have asked other SIGs to have at least two leaders.  Given the number of people interested in this topic, I would hope that it would not be difficult to find another volunteer.

Reply:
I'd also like to get someone from Netflix involved with this SIG ... going to follow-up with @cfieber.

Reply:
I'm happy to jump in here

Reply:
Done!